samples/http_example/lint.cfg

```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
We want to avoid adding bonus kokoro configurations.  If there are sample tests that need to be run here, we need to make sure the `npm run samples-test` at the root captures everything.  We probably will need to remove this :)

Reply:
![yikes](https://media2.giphy.com/media/p4cqQ0gUIMcU0/giphy.gif)

Does this mean I need to restructure the directories so all the samples are in the root?

Reply:
Of course not!  We just need to make sure that `npm run sample-tests` is actually running these :) I make a recommendation below about organizing the test files that I think will help